Sources/SwiftProtobuf/Visitor.swift

```diff
@@ -90,9 +71,11 @@ public protocol Visitor: class {
   func visitUnknown(bytes: Data)
 }
 
-/// Forwarding default implementations of some visitor methods, for convenience.
 extension Visitor {
 
+  /// Forwards to `visitRepeatedField`. Implementors who need to handle 
```

Comment:
Yup! And it renders it nicely:

![image](https://cloud.githubusercontent.com/assets/277491/22912434/22c207ce-f21a-11e6-9887-002f91eb4475.png)